flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/catalog/JavaCatalogTableTest.java

```diff
@@ -149,12 +156,119 @@ void testResolvingProctimeOfCustomTableTableApi() throws Exception {
         testUtil.verifyExecPlan(table);
     }
 
+    @TestTemplate
+    void testTimeAttributeOfView() {
+        if (!streamingMode) {
+            // time attributes not supported in batch
+            return;
+        }
+        TableTestUtil testUtil = getTestUtil();
+        TableEnvironment tableEnvironment = testUtil.getTableEnv();
+        tableEnvironment.registerCatalog("cat", new CustomCatalog("cat"));
+        tableEnvironment.executeSql(
+                "CREATE TABLE t(i INT, ts TIMESTAMP_LTZ(3), WATERMARK FOR "
+                        + "ts AS ts) WITH ('connector' = 'datagen')");
+        tableEnvironment.executeSql("CREATE VIEW `cat`.`default`.v AS SELECT * FROM t");
+        testUtil.verifyExecPlan(
+                "SELECT sum(i), window_start "
+                        + "FROM TUMBLE(\n"
```

Comment:
Seems I missed the fact it is 1.20 branch

1.20 still requires `TABLE(..)` wrapper